gensrc/script/doris_builtins_functions.py

```diff
@@ -1096,33 +1096,33 @@
     # Json functions
     [['get_json_int'], 'INT', ['VARCHAR', 'VARCHAR'],
         '_ZN5doris13JsonFunctions12get_json_intEPN9doris_udf15FunctionContextERKNS1_9StringValES6_',
-        '_ZN5doris13JsonFunctions17json_path_prepareEPN9doris_udf15FunctionContextENS2_18FunctionStateScopeE',
-        '_ZN5doris13JsonFunctions15json_path_closeEPN9doris_udf15FunctionContextENS2_18FunctionStateScopeE',
+        '_ZN5doris13JsonFunctions21json_function_prepareEPN9doris_udf15FunctionContextENS2_18FunctionStateScopeE',
```

Comment:
Modify signature may because FE-BE incompatible when upgrading. So better not changing it.